packages/main/src/SegmentedButtonItem.ts

```diff
@@ -0,0 +1,110 @@
+import customElement from "@ui5/webcomponents-base/dist/decorators/customElement.js";
+import property from "@ui5/webcomponents-base/dist/decorators/property.js";
+
+import type { I18nText } from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import { ITabbable } from "@ui5/webcomponents-base/dist/delegate/ItemNavigation.js";
+import Integer from "@ui5/webcomponents-base/dist/types/Integer.js";
+import SegmentedButtonItemTemplate from "./generated/templates/SegmentedButtonItemTemplate.lit.js";
+
+import ToggleButton from "./ToggleButton.js";
+import ButtonDesign from "./types/ButtonDesign.js";
+import Icon from "./Icon.js";
+
+// @ts-ignore
+import { SEGMENTEDBUTTONITEM_ARIA_DESCRIPTION } from "./generated/i18n/i18n-defaults.js";
+
+/**
+ * @class
+ *
+ * <h3 class="comment-api-title">Overview</h3>
+ *
+ * Users can use the <code>ui5-segmented-button-item</code> as part of a <code>ui5-segmented-button</code>.
+ * <br><br>
+ * Clicking or tapping on a <code>ui5-segmented-button-item</code> changes its state to <code>pressed</code>.
+ * The item returns to its initial state when the user clicks or taps on it again.
+ * By applying additional custom CSS-styling classes, apps can give a different style to any
+ * <code>ui5-segmented-button-item</code>.
+ *
+ * <h3>ES6 Module Import</h3>
+ *
+ * <code>import "@ui5/webcomponents/dist/SegmentedButtonItem";</code>
+ *
+ * @constructor
+ * @author SAP SE
+ * @alias sap.ui.webc.main.SegmentedButtonItem
+ * @extends sap.ui.webc.main.ToggleButton
+ * @tagname ui5-segmented-button-item
+ * @implements sap.ui.webc.main.ISegmentedButtonItem
+ * @public
+ */
+@customElement("ui5-segmented-button-item")
+class SegmentedButtonItem extends ToggleButton implements ITabbable {
```

Comment:
implements ITabbable is redundant since it comes from `ui5-button`

Reply:
Button implements " implements IFormElement" but not ITabbable?